lock/src/test/java/org/infinispan/lock/ClusteredLockTest.java

```diff
@@ -117,23 +118,23 @@ public void testLockWhenLockIsRemoved() throws Throwable {
       assertFalse(lock1Request.isDone());
       assertFalse(lock2Request.isDone());
 
-      await(cm0.remove(LOCK_NAME));
-      await(lock1Request
+      assertTrue(await(cm0.remove(LOCK_NAME)));
+      assertNull(await(lock1Request
             .exceptionally(e -> {
                assertException(ClusteredLockException.class, e);
                assertTrue(e.getMessage().contains(Log.LOCK_DELETE_MSG));
                return null;
-            }));
-      await(lock2Request
+            })));
+      assertNull(await(lock2Request
             .exceptionally(e -> {
                assertException(ClusteredLockException.class, e);
                assertTrue(e.getMessage().contains(Log.LOCK_DELETE_MSG));
                return null;
-            }));
+            })));
    }
 
    @Test
-   public void testTryLockWithTimeoutWithCountersInParallel() throws Throwable {
+   public void testNonReentrantLock() throws Throwable {
```

Comment:
Not sure why the test name got so much shorter, I'd expect it to be longer if you're adding a very similar test :)

Reply:
ok,  added very long name ;-) Hopefully more expressive now.